kanidmd/src/lib/core/mod.rs

```diff
@@ -702,6 +702,23 @@ pub async fn create_server_core(config: Configuration) -> Result<(), ()> {
 
     // Setup timed events associated to the write thread
     IntervalActor::start(server_write_ref);
+    // Setup timed events associated to the read thread
+    // TODO: only start this if enable in config.
+    // TODO: maybe we should have better option processing here
+    // and pass it on to `start_online_backup()`
+    match &config.online_backup_path {
+        Some(p) => {
+            // TODO check path exists?
```

Comment:
Should check in the event, not here :)

Reply:
Still todo, should we create the path if it is not there? I think we should try and fail if we cannot.